src/libmongoc/src/mongoc/mongoc-topology.c

```diff
@@ -413,7 +413,10 @@ mongoc_topology_new (const mongoc_uri_t *uri, bool single_threaded)
       has_directconnection &&
       mongoc_uri_get_option_as_bool (uri, MONGOC_URI_DIRECTCONNECTION, false);
    hl = mongoc_uri_get_hosts (topology->uri);
-   if (service && !has_directconnection) {
+   if (mongoc_uri_get_option_as_bool (
+          topology->uri, MONGOC_URI_LOADBALANCED, false)) {
+      init_type = MONGOC_TOPOLOGY_LOAD_BALANCED;
```

Comment:
Disabling SDAM monitoring will be in a follow-up PR. The initial type only applies until monitoring occurs. This seems sufficient to test the new sessions behavior. Starting and destroying a session does not kick off monitoring. Ending sessions does, but that behavior has not changed.